src/networks.json

```diff
@@ -211,18 +211,34 @@
   },
   "50": {
     "key": "50",
-    "name": "XinFin MainNet",
+    "name": "XDC.Network",
     "shortName": "XDC",
     "chainId": 50,
     "network": "mainnet",
-    "multicall": "",
+    "multicall": "0xc8160deb19559d93eadb82798ededce696ebcaf5",
```

Comment:
If it is possible, can you verify and publish contract code here https://xdc.blocksscan.io/address/xdcc8160deb19559d93eadb82798ededce696ebcaf5 and testnet multicall contract too

Reply:
We have verified Contracts on both explorers.